### PR TITLE
update: services

### PR DESCRIPTION
**Pull request by mb-wali**

### CODE REVIEW

File: docs/deployments/Nats.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+~ # nsc add user --account de --name services -K ABDVLFKHFWYBHJWUNC35S3VA4YGCW2SVD6MU2RSZBOJD42SJFXZLY372
+[ OK ] generated and stored user key "UAYRAHG5GXMA4DX4S46HYWWPIK5MWO64RRHXBGUDB32WXCOW6EKQFJUN"
+[ OK ] generated user creds file `/nsc/nkeys/creds/cyverse/de/services.creds`
+[ OK ] added user "services" to account "de"
+
+~ # cat /nsc/nkeys/creds/cyverse/de/services.creds
+
+```
+
+**copy secert file from the nats-box pod & create from it a secret**
```
Comment (slr71) [inline docs/deployments/Nats.md:74]: There's a typo here.

```suggestion
**copy secret file from the nats-box pod & create from it a secret**
```

File: docs/deployments/irods-csi-driver.md [hunk 2]
```diff
@@ -15,18 +17,17 @@ globalConfig:
...
       enforceProxyAccess: "true"
-      mountPathWhitelist: "/TUG/home"
+      mountPathWhitelist: "/ZONE/home"
 nodeService:
   irodsPool:
     extraArgs:
       - --cache_size_max=10737418240
-      - --cache_root=/irodsfs_pool_cache
-      - '--cache_timeout_settings=[{"path":"/","timeout":"-1ns","inherit":false},{"path":"/TUG","timeout":"-1ns","inherit":false},{"path":"/TUG/home","timeout":"5m","inherit":false},{"path":"/TUG/home/shared","timeout":"5m","inherit":true}]'
+      - '--cache_timeout_settings=[{"path":"/","timeout":"-1ns","inherit":false},{"path":"/TUG","timeout":"-1ns","inherit":false},{"path":"/TUG/home","timeout":"1h","inherit":false},{"path":"/TUG/home/shared","timeout":"1h","inherit":true}]'
```
Comment (slr71) [inline docs/deployments/irods-csi-driver.md:30]: The generic ZONE placeholder can be added here as well.

```suggestion
      - '--cache_timeout_settings=[{"path":"/","timeout":"-1ns","inherit":false},{"path":"/ZONE","timeout":"-1ns","inherit":false},{"path":"/ZONE/home","timeout":"1h","inherit":false},{"path":"/ZONE/home/shared","timeout":"1h","inherit":true}]'
```

**Review by slr71 — APPROVED** [review]
I had a couple of suggestions for cosmetic changes, but this looks good to me!

**Comment by mb-wali** [comment]
> I had a couple of suggestions for cosmetic changes, but this looks good to me!
@slr71 & @johnworth Thanks for the review -  i have pushed the changes :)

**Pull request merged**